aries-site/src/tests/utils.js

```diff
@@ -4,7 +4,7 @@ import { ReactSelector } from 'testcafe-react-selectors';
 export const baseUrl = 'http://localhost:3030';
 
 // Selector name found using Chrome React dev tools on prod mode of website
-export const Search = 'D Box';
+export const Search = 'Search__StyledTextInput';
```

Comment:
was this causing issues for you?

Reply:
Yes, when I inspected the DOM, search was no longer `D Box` but instead `B Box`, so that selector felt unstable. Changing to the specific text input to be more reliable in future changes.